src/main/java/com/liferay/faces/util/i18n/internal/I18nImpl.java

```diff
@@ -53,10 +55,27 @@ public I18nImpl() {
 
 		// Store the resource bundle cache in the application map (as a Servlet Context attribute).
 		if (startupFacesContext != null) {
+
 			ExternalContext externalContext = startupFacesContext.getExternalContext();
 			Map<String, Object> applicationMap = externalContext.getApplicationMap();
-			Map<Locale, ResourceBundle> cache = new ConcurrentHashMap<Locale, ResourceBundle>();
-			applicationMap.put(I18nImpl.class.getName(), cache);
+			Cache<Locale, ResourceBundle> facesResourceBundleCache;
+
+			String maxCacheCapacityString = externalContext.getInitParameter(I18n.class.getName() +
+					".maxCacheCapacity");
```

Comment:
@stiemannkj1 The  internal WebConfigParam class is meant to be a single source of all web.xml/portlet.xml constants in order to help identify all of the different configuration options. We need to consider the benefits/drawbacks of that approach vs a ".maxCacheCapacity" pattern. Let's discuss.